commcare_connect/opportunity/models.py

```diff
@@ -223,6 +223,23 @@ def display_name(self):
         else:
             return "---"
 
+    @property
+    def assessment_count(self):
+        return Assessment.objects.filter(user=self.user, opportunity=self.opportunity).count()
+
+    @property
+    def assessment_status(self):
```

Comment:
not blocking, but it would be a little better if we didnt have to hit the db twice. Both can be done from the same db query, but I see how this is simpler to implement.

Reply:
Added caching on assessment counts and used that property to get both total and passed/failed counts. https://github.com/dimagi/commcare-connect/pull/302/commits/4fb2f9af9e3a6749e5156814e5f59939a9362da7